src/clojure_tictactoe/game/game_rules.clj

```diff
@@ -8,10 +8,10 @@
   (apply map list (horizontal-win-options board)))
 
 (defn diagonal-win-options [board]
-  (let [side-length (board/side-length board)]
-    (let [top-left-to-bottom-right-diagonal (take side-length (range 0 (count board) (+ 1 side-length)))
-          bottom-left-to-top-right-diagonal (take side-length (range (- side-length 1) (count board) (- side-length 1)))]
-      (list top-left-to-bottom-right-diagonal bottom-left-to-top-right-diagonal))))
+  (let [side-length (board/side-length board)
+        top-left-to-bottom-right-diagonal (take side-length (range 0 (count board) (+ 1 side-length)))
+        bottom-left-to-top-right-diagonal (take side-length (range (- side-length 1) (count board) (- side-length 1)))]
+    (list top-left-to-bottom-right-diagonal bottom-left-to-top-right-diagonal)))
```

Comment:
See - https://stackoverflow.com/questions/1147975/in-clojure-when-should-i-use-a-vector-over-a-list-and-the-other-way-around

Reply:
Since I'm never accessing them outside of the order, does this matter? I am perfectly willing to change it anyway.